### ISSUE TITLE
Management interface not listed in list box for device type

### ISSUE THREAD

**Issue by mattdharris** [digitalocean/netbox#1255]
**Python version:** 3.4.5
**NetBox version:** 2.0.5

I've created two "management interfaces" on a "Console Server" type device.  The first one shows up in the list, called "Network 1" but the second which is called "Network 2" does not.  If I attempt to create "Network 2" again, I get "Network 2: Interface template with this Device type and Name already exists."  This indicates that it's there, just not showing up in the list box on the device type page.

**Comment by jeremystretch** [comment]
Does "Network 2" show up in the interfaces list? It's possible that it was created without being designated as a management interface.

**Comment by mattdharris** [comment]
Yepp, indeed it was.  Which was odd because it was a serial console so it had all the serial ports over there and I hadn't noticed it.  It was created the same way as the other mgmt interface though and that one had mgmt interface set.  
I edited it to check the 'oob mgmt' box and that put it where it belonged.  Not sure why that wasn't done right from the get go though, so still a minor bug.

**Comment by jeremystretch** [comment]
Interfaces are designed as management-only by default if you click the "add interfaces" button in the "critical connections panel" (note the `?mgmt_only=1` in the URL. Otherwise, it has to be toggled manually.

**Comment by mattdharris** [comment]
I just figured out what the issue was here after playing around with reproducing this to make sure I'm not just crazy.  I had created this via the "device type" screen, not a specific device.  It was a serial console type and not network device type, so it didn't have network ports section on the right by default.  So I clicked "create mgmt interface" from that page, and then added "Network 1", the OOB mgmt box was chcked.  I then clicked "create and add another" and it created it then on the screen to create "Network 2", it did not have that box checked.  Thus "Network 2" was created without the OOB mgmt flag set.  

When it returned to the device type definition screen, "Network 2" was nowhere to be found because the network ports on the right side didn't exist.  Going back to that screen after the fact, it appeared with "Network 2" in it.  

So this isn't really a bug then, but just something a bit counter-intuitive insofar as "create and add another" when creating a mgmt interface does not keep that box checked by default on the second and latter submissions.
